openmetadata-ui/src/main/resources/ui/src/router/AppRouter.tsx

```diff
@@ -38,6 +38,7 @@ const AppRouter = () => {
   ) : (
     <>
       {authConfig?.provider === AuthTypes.GOOGLE ||
```

Comment:
@vivekratnavel , Can we do something like this for better readability.

```
const authCheck = authConfig?.provider === AuthTypes.GOOGLE ||
      authConfig?.provider === AuthTypes.AWS_COGNITO ||
      authConfig?.provider === AuthTypes.CUSTOM_OIDC

```
```suggestion
      {authCheck ? <AuthenticatedAppRouter />:(...)
```